hack/update-generated-pod-resources.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
You're missing `hack/validate-generated-pod-resources.sh`

Reply:
added